src/asRendition.tsx

```diff
@@ -118,7 +131,7 @@ export default function asRendition<T>(
 	> = [],
 	passthroughProps: string[] = [],
 ) {
-	return compose(
+	return flowRight(
```

Comment:
I think recompose was doing it in order, in which case we should go with `flow` right? (fwiw)
See: https://github.com/acdlite/recompose/blob/3db12ce7121a050b533476958ff3d66ded1c4bb8/src/packages/recompose/compose.js#L2

Maybe we could just copy that ^ in out utils as is, so that is has a better name & we don't add lodash imports since it definitely has way more code under the hood...
See: https://github.com/lodash/lodash/blob/67389a8c78975d97505fa15aa79bec6397749807/lodash.js#L5091

Reply:
Confusing, because their docs say it's the same as lodash's `flowRight` 😂  https://github.com/acdlite/recompose/blob/3db12ce7121a050b533476958ff3d66ded1c4bb8/docs/API.md#compose

Reply:
Oh my... Another clue that ppl get confused by `.reduce`... Even them got confused by it :disappointed:

Reply:
@thgreasi I'll just copy the recompose function